### DISCUSSION TITLE
[Linux] AppImage Won't Start

### DISCUSSION THREAD

**Discussion by KingJigglypuff** [LizardByte/community#85] [Sunshine] [answered]
Since Parsec only supports Linux as a client, I looked for alternatives, which lead me here. However, I'm having a problem. I'm using the AppImage (as I don't want to install anything right now), and it just doesn't start. Though the AppRun process shows in the System Monitor, but that's it.

Is there any setup I need to do prior? Or is there something else I'm doing wrong instead.

If it helps, my Linux distro is Mint Cinnamon.

**Comment by SamuraisEpic** [comment, marked as answer]
Hmm if you're not running it from terminal try running it from terminal. (just type `./sunshine.AppImage` in the terminal in the directory you downloaded the AppImage to) otherwise, the next time you open the appimage go to localhost:47990 to verify it works. if it brings you to a login page, its running, just not telling you anything since its not running in terminal.

**Comment by KingJigglypuff** [comment]
My apologies, but I've since switched to Windows for the time being (I wanted to utilize dual booting, but did the dumb thing by installing Linux first), though whenever I get back to installing Linux on this PC, I'll be sure to give this a try.
